packages/terra-folder-tree/src/subcomponents/FolderTreeItem.module.scss

```diff
@@ -0,0 +1,30 @@
+@import '../clinical-lowlight-theme/FolderTree.module';
+@import '../orion-fusion-theme/FolderTree.module';
+
+:local {
+  .folder-tree-item {
+    align-items: center;
+    background-color: var(--terra-folder-tree-item-background-color, #fff);
+    border-bottom: var(--terra-folder-tree-item-border-bottom, 1px solid #dedfe0);
+    display: flex;
+    margin: 0;
+    min-height: var(--terra-folder-tree-item-min-height, 2.92857rem);
+    overflow-wrap: anywhere;
+    padding: 0;
+    width: 100%;
+  
+    &:hover {
+      background-color: var(--terra-folder-tree-item-hover-background-color, #f4fafe);
+    }
+  }
+  
+  .selected {
+    background-color: var(--terra-folder-tree-item-selected-background-color, darken(#ebf6fd, 2%));
+  }
+  
```

Comment:
What about the focus state?

Reply:
The focus state should be added with keyboard navigation (UXPLATFORM-9776).